examples/nidcpower_source_dc_voltage/NIDCPowerSourceDCVoltage.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.NIDCPowerSourceDCVoltage_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "MeasurementLink example that sources and measures a DC voltage with an NI SMU. Provides a Measurement UI.",
```

Comment:
"Provides a Measurement UI." is a holdover from when there were two copies of this example for measUI vs. LV UI. Please delete it here and in pyproject.toml.